### ISSUE TITLE
Pagination in API

### ISSUE THREAD

**Issue by GabrielDeveloper** [RESTBench/requirements#13]
Have some rule about pagination in API?

**Comment by Cayan** [comment]
I'm not sure about it, do you think it is need for a first spec version?

@merorafael and @igorsantos07, can you share your thoughts on this one?

**Comment by GabrielDeveloper** [comment]
@Cayan I agree, maybe in next step, but we can thinking about it right now!
